source/css/sass/_fonts.scss

```diff
@@ -8,7 +8,7 @@
   font-display: fallback;
   font-family: "Noto Sans";
   src: url("../../fonts/NotoSans-SemiBold-webfont-custom-2-subsetting.woff2") format("woff2");
-  font-weight: bold;
+  font-weight: 600;
```

Comment:
Bit confused by semi-bold being used at the bold weight.

Reply:
This is a maintenance legacy from when eLife switched typefaces a while ago. Makes sense to bump it to `600` here.